CONTRIBUTING.md

```diff
@@ -342,6 +342,23 @@ If you are a Guru, try to restrain yourself, only do magic when absolutely neces
           // Log message already printed above
       }
       ```
+    * Deliberate switch-case fall through should be explicitly indicated with a specific comment:
+      `// fall through` or `/* fall through */`
+      ```c
+      switch(x) {
+      case 1:
+      case 2:
+          // ...
+      // fall through
+      /* Optional comment explaining why */
```

Comment:
I don't think you should mix the comment styles `/* */` and `//` unless it's necessary because you want a multiline comment. So in this case, I would:

```suggestion
      // Optional comment explaining why
```

Reply:
I put the `/* */` to imply that it may be a multiline comment. It didnt come out well. Will fix along side Vratislavs change request